### PR TITLE
TOMEE-2569 - translation of cdi-interceptor example.

### PR DESCRIPTION
**Pull request by gabrielferreirapro**
Hey guys there is my translation of cdi-application-scope example.
Thank you ;)
jira task: https://issues.apache.org/jira/browse/TOMEE-2569

### CODE REVIEW

File: examples/cdi-interceptors/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+ @Retention(RUNTIME)
+ public @interface TimeRestricted {
+ }
+
+Garanta que voce não esqueceu a anotação `@InterceptorBinding` acima! Agora nossas anotações personalizadas estão criadas, vamos anexa-las (ou vincula-las) aos interceptores.
+
+Aqui esta nosso interceptor de log. Um método `@AroundInvoke` e estamos quase terminando.
+
+ @Interceptor
+ @Log  //binding the interceptor here. now any method annotated with @Log would be intercepted by logMethodEntry
```
Comment (Daniel-Dos) [inline examples/cdi-interceptors/README_pt.adoc]: translate this comment :  )

Comment (gabrielferreirapro) [reply examples/cdi-interceptors/README_pt.adoc]: Thank you @Daniel-Dos, revised.

**Comment by Daniel-Dos** [comment]
Hi @gabrielferreirapro , 

I not found an e-mail about this PR in list of dev-tomee .

**Comment by gabrielferreirapro** [comment]
> Hi @gabrielferreirapro ,
> 
> I not found an e-mail about this PR in list of dev-tomee .

It's true, I forgot... my bad.

**Pull request merged**